custom.scss

```diff
@@ -2,6 +2,25 @@
 
 /*-- scss:rules --*/
 
+$module-bg-color: rgba(188, 188, 254, 0.25);
+$module-bg-color-highlight: rgba(188, 188, 254, 0.75);
+
+
+/* Prevent anchor links from overlapping with header:
+   https://stackoverflow.com/questions/4086107/fixed-page-header-overlaps-in-page-anchors
+*/
+html {
+  scroll-padding-top: 85px;
+}
+
+
+/* Override built-in style which adds space after nested list elements */
+ul ul, ol ol, ul ol, ol ul {
+  margin-bottom: 0px !important;
+}
```

Comment:
The old style had a bottom-margin on nested lists, so e.g.

* Foo
    * Bar
* Baz

Would display as 

* Foo
    * Bar

* Baz